src/types.ts

```diff
@@ -2241,6 +2241,7 @@ export type AppSettings = {
   disable_permissions_checks?: boolean;
   enforce_unique_usernames?: 'no' | 'app' | 'team';
   event_hooks?: Array<EventHook> | null;
+  explicit_hook_v2_deletion?: boolean;
```

Comment:
Is it necessary to include v2 in the name?

Reply:
Yeah, I think its better to rename to `explicit_event_hooks_deletion` because it relates to it.